components/connection-token-handler/pkg/controller/tokenrequest/tokenrequest_client_test.go

```diff
@@ -22,29 +22,55 @@ func getServerMock(t *testing.T, appName, token string) *httptest.Server {
 		}),
 	)
 }
+
+func checkHeaders(t *testing.T, appName string, r *http.Request, tenant string, group string) {
+	assert.Equal(t, appName, r.Header.Get(applicationHeader))
+	if tenant != "" && group != "" {
+		assert.Equal(t, tenant, r.Header.Get(tenantHeader))
+		assert.Equal(t, group, r.Header.Get(groupHeader))
+	}
+}
+
 func TestTokenRequestClient_FetchToken(t *testing.T) {
 	appName := "some-app"
 	token := "some-long-token-value"
 
 	t.Run("should return TokenDto with valid token", func(t *testing.T) {
-		srvMock := getServerMock(t, appName, token)
+		//given
+		srvMock := getServerMock(t, appName, "", "", token)
```

Comment:
Please move those "" to some variables with descriptive name (eg. emptyTenant, emptyGroup). Currently, without checking the method definition it is hard to figure out what is what.

Reply:
Done